### ISSUE TITLE
[BUG] .fui-row-empty class not applied on row with only hidden field

### ISSUE THREAD

**Issue by Antimated** [verbb/formie#474]
**Description**
iirc there was a fui-row-empty class (or something to the likes of that) being added on rows where no visible items are being rendered. Now this is not the case anymore

![fui-row-empty](https://user-images.githubusercontent.com/2869409/119808202-2fe09380-bee4-11eb-9f8d-c33efe264d9c.jpeg)

**Steps to reproduce**
1. Create a formie form
2. Add a hidden field
3. Render form and look at outputted html

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.4
- Craft version: Craft Pro 3.6.15
- Multi-site: No

**Comment by engram-design** [comment]
This was actually removed in [1.4.0](https://github.com/verbb/formie/releases/tag/1.4.0) because we no longer style the `fui-row` class with top/bottom margin. However, I imagine you're styling that with your custom solution, so we've added this back - albeit we're not bundling any styling with this. It's just there if you need to style the class yourself.

Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.5](https://github.com/verbb/formie/releases/tag/1.4.5)
